src/layers/legacy/medCoreLegacy/gui/medAbstractWorkspaceLegacy.cpp

```diff
@@ -171,13 +170,26 @@ medTabbedViewContainers* medAbstractWorkspaceLegacy::tabbedViewContainers() cons
     return d->viewContainerStack;
 }
 
-void medAbstractWorkspaceLegacy::clear()
```

Comment:
it was not used when the user switches between a workspace to another?

Reply:
no